### ISSUE TITLE
IPify Error - TLS CA certificate bundle

### ISSUE THREAD

**Issue by dwasss** [johnnykv/heralding#61]
I set the option to get the public IP from IPify in the Heralding config. When starting Heralding for the first time, there are no issues but every hour after that I see errors like the one below.

`2017-08-31 14:09:40,082 (heralding.honeypot) Could not request public ip from ipify, error: Could not find a suitable TLS CA certificate bundle, invalid path: /home/<user>/.local/lib/python3.5/site-packages/certifi/cacert.pem`

I checked and that path and file are valid:
```
$ ls -ltrha /home/<user>/.local/lib/python3.5/site-packages/certifi/cacert.pem
-rw-rw-r-- 1 <user> <user> 290K Aug 29 17:30 /home/<user>/.local/lib/python3.5/site-packages/certifi/cacert.pem
```

Any ideas?

**Comment by kajojify** [comment]
@dwasss, Hello!
Unfortunately, I couldn't reproduce an issue on my system. 
After short googling I found a few repositories with the same issue. It seems, this issue goes from ez_setup.py file.
I am not sure, it will work for you, but there are two solution I found:
- second part of https://github.com/maxtepkeev/python-redmine/issues/167#issue-213598408
- https://github.com/cs50/submit50/issues/75#issuecomment-307590933
